bullet_train-themes-light/postcss-import-config.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+const { execSync } = require("child_process");
+
+const themeStylesheet = execSync(`bundle exec bin/theme tailwind-stylesheet ${process.env.THEME} 2> /dev/null`).toString().trim()
+const themeRoot = execSync(`bundle show bullet_train-themes-${process.env.THEME} 2> /dev/null`).toString().trim()
+const themeStylesheetsDir = path.resolve(themeRoot, 'app/assets/stylesheets/');
```

Comment:
What do you think if we do this?

```js
const gemList = execSync(`bundle list --name-only 2> /dev/null`).toString().trim()
const gemName = gemList.split("\n").find((gem) => gem == `bullet_train-themes-${process.env.THEME}`)
let themeRoot = ""
if(gemName != undefined){
  const themeRoot = execSync(`bundle show bullet_train-themes-${process.env.THEME} 2> /dev/null`).toString().trim()
}
const themeStylesheetsDir = path.resolve(themeRoot, 'app/assets/stylesheets/');
```

I found that the child process would quit `bin/dev` if `bundle show` didn't find what we were looking for, so using `bundle list` seems like a good way to find the information we want.

Reply:
@gazayas Let me know what you think of this alternative I pushed: I've added `bin/theme stylesheets-dir {THEME}` to return the stylesheets path for the gem. In the case of the light theme, it would return `{gem_path}/app/assets/stylesheets/light/`

https://github.com/bullet-train-co/bullet_train-core/blob/aff1b3f7996700f5d03fd8a0610110c2647b2c10/bullet_train-themes-light/postcss-import-config.js#L3-L4

And so `light.tailwind.css` will just need to import:

https://github.com/bullet-train-co/bullet_train-core/blob/aff1b3f7996700f5d03fd8a0610110c2647b2c10/bullet_train-themes-light/app/assets/stylesheets/light.tailwind.css#L5

Reply:
@pascallaliberte LGTM! I confirmed that things are working properly against the default app, and when a custom theme is ejected.